pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -287,3 +291,10 @@ def _should_track_batch_outputs_for_epoch_end(self) -> bool:
         if self.trainer.testing:
             return is_overridden("test_epoch_end", model)
         return is_overridden("validation_epoch_end", model)
+
+    def _initialize_batch_idx_tracker(self, num_dataloaders: int) -> None:
+        self._seen_batches = [0] * num_dataloaders
```

Comment:
shouldn't this initialize with the existing ptracking value in case of reload?

Reply:
what if the number of dataloaders changes? if they are new dataloaders, I'd suggest reinitializing them